suse/rdma-core.spec

```diff
@@ -0,0 +1,698 @@
+Obsoletes:      libibverbs-devel < %{version}-%{release}
+
+Provides:       libibcm-devel = %{version}-%{release}
+Obsoletes:      libibcm-devel < %{version}-%{release}
+
+Provides:       libibumad-devel = %{version}-%{release}
+Obsoletes:      libibumad-devel < %{version}-%{release}
+Provides:       librdmacm-devel = %{version}-%{release}
+
+Obsoletes:      librdmacm-devel < %{version}-%{release}
+#Requires:       ibacm = %%{version}-%%{release}
+Provides:       ibacm-devel = %{version}-%{release}
+Obsoletes:      ibacm-devel < %{version}-%{release}
+
+%description devel
+RDMA core development libraries and headers.
+
+%package -n     libibverbs
+Summary:        Library & drivers for direct userspace use of InfiniBand/iWARP/RoCE hardware
+Group:          System/Libraries
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Provides:       libcxgb3-rdmav2 = %{version}-%{release}
+Obsoletes:      libcxgb3-rdmav2 < %{version}-%{release}
+Provides:       libcxgb4-rdmav2 = %{version}-%{release}
+Obsoletes:      libcxgb4-rdmav2 < %{version}-%{release}
+Provides:       libhfi1verbs-rdmav2 = %{version}-%{release}
+Obsoletes:      libhfi1verbs-rdmav2 < %{version}-%{release}
+Provides:       libi40iw-rdmav2 = %{version}-%{release}
+Obsoletes:      libi40iw-rdmav2 < %{version}-%{release}
+Provides:       libipathverbs-rdmav2 = %{version}-%{release}
+Obsoletes:      libipathverbs-rdmav2 < %{version}-%{release}
+Provides:       libmlx4-rdmav2 = %{version}-%{release}
+Obsoletes:      libmlx4-rdmav2 < %{version}-%{release}
+Provides:       libmlx5-rdmav2 = %{version}-%{release}
+Obsoletes:      libmlx5-rdmav2 < %{version}-%{release}
+Provides:       libmthca-rdmav2 = %{version}-%{release}
+Obsoletes:      libmthca-rdmav2 < %{version}-%{release}
+Provides:       libnes-rdmav2 = %{version}-%{release}
+Obsoletes:      libnes-rdmav2 < %{version}-%{release}
+Provides:       libocrdma-rdmav2 = %{version}-%{release}
+Obsoletes:      libocrdma-rdmav2 < %{version}-%{release}
+Provides:       librxe-rdmav2 = %{version}-%{release}
+Obsoletes:      librxe-rdmav2 < %{version}-%{release}
+%if 0%{?dma_coherent}
+Requires:       %{mlx4_lname} = %{version}-%{release}
+Requires:       %{mlx5_lname} = %{version}-%{release}
+%endif
+
+%description -n libibverbs
+libibverbs is a library that allows userspace processes to use RDMA
+"verbs" as described in the InfiniBand Architecture Specification and
+the RDMA Protocol Verbs Specification.  This includes direct hardware
+access from userspace to InfiniBand/iWARP adapters (kernel bypass) for
+fast path operations.
+
+Device-specific plug-in ibverbs userspace drivers are included:
+
+- libcxgb3: Chelsio T3 iWARP HCA
+- libcxgb4: Chelsio T4 iWARP HCA
+- libhfi1: Intel Omni-Path HFI
+- libhns: HiSilicon Hip06 SoC
+- libi40iw: Intel Ethernet Connection X722 RDMA
+- libipathverbs: QLogic InfiniPath HCA
+- libmlx4: Mellanox ConnectX-3 InfiniBand HCA
+- libmlx5: Mellanox Connect-IB/X-4+ InfiniBand HCA
```

Comment:
The descriptions are a direct copy of the redhat spec file. It might be worth fixing  both specs in a later patch.